app/views/providers/allocations/pick_a_provider.html.erb

```diff
@@ -0,0 +1,65 @@
+<%= content_for :page_title, "Pick a provider"%>
+
+<% content_for :before_content do %>
+  <%= govuk_back_link_to(initial_request_provider_recruitment_cycle_allocations_path(@provider.provider_code, @recruitment_cycle.year)) %>
+<% end %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-xl" data-qa="heading">Pick a provider</h1>
+
+    <p class="govuk-body">You searched for ‘<%= params[:training_provider_query] %>’</p>
+    <p class="govuk-body">We found these providers which matched your search:</p>
+
+    <ul class="govuk-list govuk-list--bullet">
+      <% training_providers.each do |training_provider| %>
+        <li data-qa="provider_suggestion">
+          <%= govuk_link_to "#{training_provider.provider_name} (#{training_provider.provider_code})",
+            initial_request_provider_recruitment_cycle_allocations_path(training_provider_code: training_provider.provider_code), { class: "govuk-link" }
```

Comment:
I don't think we need to declare the css class, it should be the default for this helper method.

Reply:
It's from a copy and paste but yes it's not needed and has now been removed